pkg/controller/dataimportcron-controller.go

```diff
@@ -640,6 +640,7 @@ func (r *DataImportCronReconciler) newCronJob(cron *cdiv1.DataImportCron) (*v1be
 
 	var successfulJobsHistoryLimit int32 = 0
 	var failedJobsHistoryLimit int32 = 0
+	var activeDeadlineSeconds int64 = 120
```

Comment:
So the job just updates the digest on the DIC right? And this would put a limit of 2 minutes on it succeeding with that?